Resources/Prototypes/_Moffstation/Entities/Objects/Weapons/Guns/Ammunition/Boxes/light_rifle.yml

```
@@ -0,0 +1,18 @@
+# Moffstation - Begin - Apply directly to vampires and/or werewolves.
+- type: entity
+  parent: BaseMagazineBoxLightRifle
+  id: MagazineBoxLightRifleSilver
+  name: ammunition box (.30 rifle silver)
+  description: A cardboard box of .30 rifle rounds. Intended to hold holy silver ammunition.
+  components:
+  - type: BallisticAmmoProvider
+    proto: CartridgeLightRifleSilver
+  - type: Sprite
+    sprite: _Moffstation/Objects/Weapons/Guns/Ammunition/Boxes/light_rifle.rsi
+    layers: 
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-1
+      map: ["enum.GunVisualLayers.Mag"]
+    - state: silver
+# Moffstation - End
```

```suggestion
- type: entity
  parent: BaseMagazineBoxLightRifle
  id: MagazineBoxLightRifleSilver
  name: ammunition box (.30 rifle silver)
  description: A cardboard box of .30 rifle rounds. Intended to hold holy silver ammunition.
  components:
  - type: BallisticAmmoProvider
    proto: CartridgeLightRifleSilver
  - type: Sprite
    sprite: _Moffstation/Objects/Weapons/Guns/Ammunition/Boxes/light_rifle.rsi
    layers: 
    - state: base
      map: ["enum.GunVisualLayers.Base"]
    - state: mag-1
      map: ["enum.GunVisualLayers.Mag"]
    - state: silver
```
No moff guards in `_moff` directories.